changelog.txt

```diff
@@ -1,3 +1,6 @@
+0.81.1
+ * Fixed a memory leak issue with the FinalizerRunnable (PhantomReference)
```

Comment:
This is very internal. Perhaps rephrase to "Fixed memory leak causing Realm to never release Realm objects." or something that would make sense from a user stand point.